src/main/java/io/github/tjheslin1/patterdale/http/MetricsServlet.java

```diff
@@ -56,7 +56,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
         formatProbeResults(probeResults)
                 .forEach(formattedProbeResult -> {
                     try {
-                        resp.getWriter().println(formattedProbeResult);
+                        resp.getWriter().print(formattedProbeResult + "\n");
```

Comment:
Could "\n" be replaced with `System.lineSeparator()` ? 🙏

Reply:
System.lineSeparator()  returns "\r\n" on windows which would lead to the same result as println

Reply:
Ah I see.